pkg/cmd/attestation/verification/attestation.go

```diff
@@ -113,3 +114,27 @@ func GetRemoteAttestations(c FetchAttestationsConfig) ([]*api.Attestation, error
 	}
 	return nil, fmt.Errorf("owner or repo must be provided")
 }
+
+type DssePayload struct {
+	PredicateType string `json:"predicateType"`
+}
+
+func FilterAttestations(predicateType string, attestations []*api.Attestation) []*api.Attestation {
```

Comment:
What do you think of adding a few unit tests for the new `FilterAttestations` function? Otherwise, looks good to me.